home-work3/server.js

```diff
@@ -0,0 +1,60 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const app = express();
+const PORT = 3000;
+
+const {
+    getEvents,
+    getEventsByLocation,
+    getEventById,
+    addEvent,
+    updateEvent
+} = require('./utils');
+
+app.use(bodyParser.json());
+
+app.get('/events-batch', async (req, res) => {
+    const events = await getEvents();
+    res.json(events);
+});
+
+app.get('/events', async (req, res) => {
+    const {location} = req.query;
+
+    if (!location) {
```

Comment:
Potentially `location` could be not set. We would like to still get all events in such case.